pom.xml

```diff
@@ -34,6 +34,7 @@
   <modules>
     <module>compiler</module>
     <module>core</module>
+    <module>gwt</module>
```

Comment:
Doesn't it need to be added in the `examples` profile as well?

(Also, providing an example --just duplicating the simple one and adapting it to GWT, to begin with-- could serve as an integration test)